code/modules/antagonists/wizard/equipment/artefact.dm

```diff
@@ -110,6 +110,19 @@
 	eat()
 	return
 
+/obj/singularity/wizard/attack_tk(mob/user)
+	if(iscarbon(user))
+		var/mob/living/carbon/C = user
+		to_chat(C, "<span class='userdanger'>OH GOD! NONE OF ITS REAL! NONE OF IT IS REEEEEEEEEEEEEEEEEEEEEEEEAL!</span>")
+		//for(var/datum/component/mood/insaneinthemembrane in C.datum_components)
+		//	insaneinthemembrane.sanity = 0
+		for(var/lore in typesof(/datum/brain_trauma/severe))

```

Comment:
You can only get three traumas at a time? Any way to change this variable just for the 10 seconds the victim spends alive?

Reply:
tbh i don't know why you bother with the traumas if you also kill them right after